cms/static/js/certificates/models/certificate.js

```diff
@@ -20,9 +20,25 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             name: 'Name of the certificate',
             description: 'Description of the certificate',
-            course_title: 'Title of the course',
+
+            /**
+             * It represents the course title.
```

Comment:
@catong yes, we are showing this text via placeholder attribute in field. so i just removed default text from model.